scripts/BuildMultitoolForNpm.ps1

```diff
@@ -0,0 +1,75 @@
+<#
+.SYNOPSIS
+    Build, and package the Sarif Multitool NPM package.
+.DESCRIPTION
+    Builds the Sarif Multitool NPM package, including building the .NET Core 3.0 single-file-exe of the Multitool for all supported platforms.
+.PARAMETER Configuration
+    The build configuration: Release or Debug. Default=Release
+#>
+
+[CmdletBinding()]
+param(
+    [string]
+    [ValidateSet("Debug", "Release")]
+    $Configuration="Release",
+
+    [switch]
+    $SkipBuild,
+
+    [switch]
+    $NoPostClean
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+$InformationPreference = "Continue"
+
+$ScriptName = $([io.Path]::GetFileNameWithoutExtension($PSCommandPath))
+
+Import-Module -Force $PSScriptRoot\ScriptUtilities.psm1
+Import-Module -Force $PSScriptRoot\NuGetUtilities.psm1
+Import-Module -Force $PSScriptRoot\Projects.psm1
+
+$project = "Sarif.Multitool"
+$projectBinDirectory = (Get-ProjectBinDirectory $project $Configuration)
+$npmSourceFolder = "$RepoRoot\npm"
+$npmBuildFolder = "$BuildRoot\Publish\npm"
+
+if (-not $SkipBuild) {
+    Write-Information "Building Sarif.Multitool for Windows, Linux, and MacOS..."
+    dotnet publish $SourceRoot\$project\$project.csproj -c $Configuration -f netcoreapp3.0 -r win-x64
+    dotnet publish $SourceRoot\$project\$project.csproj -c $Configuration -f netcoreapp3.0 -r linux-x64
+    dotnet publish $SourceRoot\$project\$project.csproj -c $Configuration -f netcoreapp3.0 -r osx-x64
+
+    Write-Information "Merging binaries [$projectBinDirectory] and NPM configuration [$npmSourceFolder]..."
+    New-DirectorySafely $npmBuildFolder\
+    Copy-Item -Force -Container -Recurse -Path $npmSourceFolder\* -Destination $npmBuildFolder\
+    Copy-Item -Force -Container -Recurse -Path $projectBinDirectory\Publish\netcoreapp3.0\win-x64\* -Destination $npmBuildFolder\sarif-multitool-win32\
+    Copy-Item -Force -Container -Recurse -Path $projectBinDirectory\Publish\netcoreapp3.0\linux-x64\* -Destination $npmBuildFolder\sarif-multitool-linux\
+    Copy-Item -Force -Container -Recurse -Path $projectBinDirectory\Publish\netcoreapp3.0\osx-x64\* -Destination $npmBuildFolder\sarif-multitool-darwin\
+}
+
+# TEMP: Use 0.x.y numbering until NPM package shape settled, then follow SDK version exactly by removing the hardcoded set
+$sarifVersion = Get-PackageVersion
+$sarifVersion = "0.1.25"
+
+Write-Information "Injecting Sarif SDK version $sarifVersion for NPM Packages..."
+foreach ($package in (Get-ChildItem $npmBuildFolder).FullName) {
+    Find-AndReplaceInFile "$package\package.json" "0\.1\.0" $sarifVersion
```

Comment:
>0\.1\.0 [](start = 51, length = 7)

This assumes that `"0.1.0"` appears nowhere else in the file (for example as the version number of some other dependency). That's probably safe, but you could choose instead to use some other token, like `$SarifSdkVersion`, in the package.json files.

Reply:
Good suggestion. Settled on {version} to not interfere with special symbols in PowerShell ($) or Regex.